src/kvstore/DiskManager.cpp

```diff
@@ -114,6 +114,14 @@ void DiskManager::removePartFromPath(GraphSpaceID spaceId,
     partPath_[spaceId][canonical].erase(partId);
 }
 
+StatusOr<PartDiskMap> DiskManager::partDist(GraphSpaceID spaceId) {
+    auto spaceIt = partPath_.find(spaceId);
+    if (spaceIt == partPath_.end()) {
+        return Status::Error("Space not found");
+    }
+    return spaceIt->second;
+}
+
```

Comment:
Where is this method called?

Reply:
> Where is this method called?

No where for now, prepared for later vertical balance.